datasets/mt_eng_vietnamese/mt_eng_vietnamese.py

```diff
@@ -0,0 +1,127 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import collections
+
+import datasets
+
+
+_DESCRIPTION = """\
+Preprocessed Dataset from IWSLT'15 English-Vietnamese machine translation: English-Vietnamese.
+"""
+
+_CITATION = """\
+@inproceedings{Luong-Manning:iwslt15,
+        Address = {Da Nang, Vietnam}
+        Author = {Luong, Minh-Thang  and Manning, Christopher D.},
+        Booktitle = {International Workshop on Spoken Language Translation},
+        Title = {Stanford Neural Machine Translation Systems for Spoken Language Domain},
+        Year = {2015}}
+"""
+
+_DATA_URL = "https://nlp.stanford.edu/projects/nmt/data/iwslt15.en-vi/{}.{}"
+
+# Tuple that describes a single pair of files with matching translations.
+# language_to_file is the map from language (2 letter string: example 'en')
+# to the file path in the extracted directory.
+TranslateData = collections.namedtuple("TranslateData", ["url", "language_to_file"])
+
+
+class MT_Eng_ViConfig(datasets.BuilderConfig):
+    """BuilderConfig for MT_Eng_Vietnamese."""
+
+    def __init__(self, language_pair=(None, None), **kwargs):
+        """BuilderConfig for MT_Eng_Vi.
+        Args:
+            for the `datasets.features.text.TextEncoder` used for the features feature.
+          language_pair: pair of languages that will be used for translation. Should
+            contain 2-letter coded strings. First will be used at source and second
+            as target in supervised mode. For example: ("vi", "en").
+          **kwargs: keyword arguments forwarded to super.
+        """
+
+        description = ("Translation dataset from %s to %s") % (language_pair[0], language_pair[1])
+        super(MT_Eng_ViConfig, self).__init__(
+            description=description,
+            version=datasets.Version("1.0.0"),
+            **kwargs,
+        )
+        self.language_pair = language_pair
+
+
+class MT_Eng_Vietnamese(datasets.GeneratorBasedBuilder):
+    """English Vietnamese machine translation dataset from IWSLT2015."""
+
+    BUILDER_CONFIGS = [
+        MT_Eng_ViConfig(
+            name="iwslt2015-vi-en",
+            language_pair=("vi", "en"),
+        ),
+        MT_Eng_ViConfig(
+            name="iwslt2015-en-vi",
+            language_pair=("en", "vi"),
+        ),
+    ]
```

Comment:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = MT_Eng_ViConfig
```
we need that to specify that we're using a custom config